src/api/control/async/asyncMotionProfileController.cpp

```diff
@@ -253,23 +253,25 @@ void AsyncMotionProfileController::executeSinglePath(const TrajectoryPair &path,
 
   if (followMirrored) {
     for (int i = 0; i < path.length && !isDisabled(); ++i) {
+      float segDT = path.left[i].dt;
```

Comment:
So I guess we assume that the left and right sides have the same dt? Seems reasonable to me but I could be persuaded against this.

Reply:
I mean, unless you want to run a different task for each side. I think it's a totally reasonable assumption to make- Pathfinder only generates paths that have a constant DT that's the same for both sides. It would be a massive rewrite that's far more complex for no benefit.